modules/connection.js

```diff
@@ -0,0 +1,22 @@
+var mysql = require('mysql');
+
+var connection = mysql.createConnection({
+	host     : 'localhost',
+	user     : 'root',
+	password : '',
+	database : 'meete_APP'
+});
+ 
+connection.connect(function(err){
+	if (err) {
+		var error = {
+			status: 0,
+			message: "Error in execution"
+		}
```

Comment:
what will happen if there is an error

Reply:
it will show msg 
status: 0,	
message: "Error in execution"
in the console

Reply:
?